mobile/src/Student/screens/Game/PhaseTwoBasicGamePlay/index.js

```diff
@@ -23,6 +23,11 @@ import HorizontalPageView from "../../../components/HorizontalPageView"
 import ScrollableQuestion from "../Components/ScrollableQuestion"
 import AnswerOptionsPhaseTwo from "./AnswerOptionsPhaseTwo"
 import HintsView from "../Components/HintsView"
+import RoundTextIcon from "../../../components/RoundTextIcon"
+
+const indexToLetter = (index) => {
+    return String.fromCharCode(65 + index)
```

Comment:
Nice find, if you want to go this path, some comments on the code would be useful for future you and others reading the code.

Reply:
Sounds good! Will add some commenting.